bukuserver/views.py

```diff
@@ -40,34 +40,20 @@ class CustomAdminIndexView(AdminIndexView):
     def index(self):
         return self.render("bukuserver/home.html", form=forms.HomeForm())
 
-    @expose(
-        "/",
-        methods=[
-            "POST",
-        ],
-    )
+    @expose('/', methods=['POST'])
     def search(self):
         "redirect to bookmark search"
         form = forms.HomeForm()
-        bbm_filter = bs_filters.BookmarkBukuFilter(
-            all_keywords=False, deep=form.deep.data, regex=form.regex.data
-        )
-        op_text = bbm_filter.operation()
-        values_combi = sorted(itertools.product([True, False], repeat=3))
-        choosen_idx = None
-        for idx, (all_keywords, deep, regex) in enumerate(values_combi):
-            if deep == form.deep.data and regex == form.regex.data and not all_keywords:
-                choosen_idx = idx
-        url_op_text = op_text.replace(", ", "_").replace("  ", " ").replace(" ", "_")
-        kwargs = {}
-        if choosen_idx:
-            key = "".join(["flt", str(choosen_idx), "_buku_", url_op_text])
-            kwargs = {key: form.keyword.data}
-        url = url_for("bookmark.index_view", **kwargs)
+        buku_filter = bs_filters.BookmarkBukuFilter(deep=form.deep.data, regex=form.regex.data)
+        url = url_for('bookmark.index_view', **{filter_key(buku_filter): form.keyword.data})
         return redirect(url)
 
 
 class BookmarkModelView(BaseModelView):
+    @staticmethod
+    def _filter_arg(flt):  # this works because BookmarkModelView.named_filter_urls = True
```

Comment:
change this to function comment rather than single line comment

Reply:
…Are you talking about a docstring (placed on the next line) or a comment on its own line (placed before `@staticmethod` I guess)?

Reply:
…Converted to a docstring for now.

Reply:
Yes, function docstring